<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicConnectTransferWidget</class>
 <widget class="QWidget" name="MusicConnectTransferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>传文件到移动设备</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>620</width>
     <height>369</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>传文件到移动设备</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>620</width>
     <height>344</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(255,255,255,150);}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QWidget" name="playListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>0</y>
      <width>361</width>
      <height>344</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,255,255,50)</string>
    </property>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>5</y>
       <width>165</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="allSelectedcheckBox">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>10</y>
       <width>95</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchLineLabel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>5</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/share/searchlineleft);</string>
     </property>
    </widget>
    <widget class="MusicConnectTransferTableWidget" name="playListTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>361</width>
       <height>304</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="functionBottomLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>361</width>
       <height>304</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255,255,255)</string>
     </property>
    </widget>
    <zorder>searchLineEdit</zorder>
    <zorder>allSelectedcheckBox</zorder>
    <zorder>searchLineLabel</zorder>
    <zorder>functionBottomLabel</zorder>
    <zorder>playListTableWidget</zorder>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>40</y>
      <width>145</width>
      <height>304</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mobileModeWidget">
     <widget class="QLabel" name="iconUSBLabel">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>50</y>
        <width>72</width>
        <height>72</height>
       </rect>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../MMMQrc/MusicPlayer.qrc">:/toolSets/usb</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="textUSBLabel">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>120</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="transferUSBButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>transfer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reflashUSBButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>reflash</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wifiModeWidget">
     <widget class="QLabel" name="iconWIFILabel">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>50</y>
        <width>72</width>
        <height>72</height>
       </rect>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../MMMQrc/MusicPlayer.qrc">:/toolSets/wifi</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="transferWIFIButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>transfer</string>
      </property>
     </widget>
     <widget class="QLabel" name="WIFILabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>120</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>对方IP地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineWIFIEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>wifi</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicConnectTransferTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicconnecttransfertablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../MMMQrc/MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
